clusters/sandbox-values.yaml

```diff
@@ -5,11 +5,17 @@ global:
     ip: "10.101.29.134"
 
 namespaces:
+- name: sandbox-connector-node-metadata
+  owner: alphagov
+  repository: verify-metadata
+  branch: master
+  path: ci
```

Comment:
This path doesn't appear to exist in [the repo](https://github.com/alphagov/verify-metadata) on `master`.

Reply:
No, that's the next thing I'm doing.